packages/sodium/lib/src/api/secure_key_extensions.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:sodium/sodium.dart';
```

Comment:
Please do not import the root library in src files. Instead, import everything you need via relative imports.

Reply:
Done.